crates/re_types_core/src/lib.rs

```diff
@@ -20,6 +20,9 @@
 
 // ---
 
+/// Number of decimals shown for all float display methods.
+pub const DISPLAY_PRECISION: usize = 3;
```

Comment:
```suggestion
pub const DEFAULT_DISPLAY_DECIMALS: usize = 3;
```

with the other changes